src/Microsoft.ML/Data/TextLoader.cs

```diff
@@ -0,0 +1,160 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Api;
+using Microsoft.ML.Runtime.Data;
+using System;
+using System.Linq;
+using System.Reflection;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.ML.Data
+{
+    public sealed partial class TextLoaderRange
+    {
+        public TextLoaderRange()
+        {
+        }
+
+        public TextLoaderRange(int ordinal)
+        {
+            Min = ordinal;
+            Max = ordinal;
+        }
```

Comment:
You may want to document for the users what the purpose of this is. It may not be obvious.

Reply:
Something like, perhaps, `Convenience constructor for the scalar case. Min and Max are set to the single value ordinal." (Naturally use proper `<see` and `<paramref` cases.)

I'm writing that 

---
In reply to: [188720218](https://github.com/dotnet/machinelearning/pull/142#discussion_r188720218) [](ancestors = 188720218)